### DISCUSSION TITLE
Duty cycle management

### DISCUSSION THREAD

**Discussion by Adamzat** [Lora-net/LoRaMac-node#1140] [Q&A] [answered]
Hi all, 

I updated my device EU868 to LoraMac 4.5.1. The Uplinks and Downlinks work well.

However, I have a problem when switching OFF the duty cycle for tests. Actually, when the Duty cycle is OFF, the Rejoin or Uplinks are sent immediately (I mean It works fine), but the device stops sending after about 30 minutes. Uplinks are possible again after 90 minutes.

After, the Debug, I observed that the LoraMac status is switched to “LORAMAC_STATUS_DUTYCYCLE_RESTRICTED”. 
Is there a way to fix this problem? If I am not wrong, if we disable the Duty cycle we should never enter in this status.

Thank you for you replay.

**Comment by mluis1** [comment, marked as answer]
This has been fixed in version [v4.5.2](https://github.com/Lora-net/LoRaMac-node/releases/tag/v4.5.2) release.
The commit fixing it is 691086af5524803d17fd22d3096cda62da4b0713

I would recommend to update your code base to the latest available version.

In the future it would be nice if you could post this kind of questions on the project [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab. It is a better place to engage discussions and then we can agree if it is an issue or not.
If you don't mind I move this issue to the [Discussions](https://github.com/Lora-net/LoRaMac-node/discussions) tab.

**Reply by Adamzat** [reply]
Hi, 
I updated to 4.5.2. The Bug is still observed during the Rejoin (not for uplinks). 
Regards.

**Reply by mluis1** [reply]
The provided fix only applies if the end-device is powered OFF/ON or reset.

I would recommend that you read LoRaWAN 1.0.4 specification chapter 7 "Retransmissions Backoff".
In this chapter you will read that the retransmission backoff uses the time the end-device was powered ON as reference for the restrictions computation.
![image](https://user-images.githubusercontent.com/5763295/127307847-773e225e-4a84-478d-9c6e-63bffd9860a2.png)

The current behavior implementation is in accordance to the LoRaWAN specification.

**Reply by Adamzat** [reply]
It is clear Thank you
